news/2025/2025-12-12-osumania-7k-world-cup-2026-registrations-now-open.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2026: Registrations Now Open!"
+date: 2025-12-12 12:00:00 +0000
+series: world_cups
+---
+
+Key smashing awesomeness is once again upon us. It's the 10th edition of the osu!mania 7K World Cup!
+
+![](/wiki/Tournaments/MWC/2026_7K/img/mwc7k2026-banner.jpg)
+
+As 2026 approaches, we are updating the tournament's design language (which looks amazing, by the way) and tweaking the format. Here is what we have in store:
```

Comment:
if you're gonna mention the art then a smol quick artist credit would be cool, please and ty 🙏
as is this reads like a slight bit of self-praise

Reply:
it was planned I just forgot to finish that portion :x